index.html

```diff
@@ -82,203 +79,166 @@ <h5>Multi-Post Stories</h5>
-            <li>Skills</li>
-          </ul>
-        </div>
-        <div class="frame2">
-          <ul>
-            <li>Database Management</li>
-            <li>Version Control</li>
-            <li>CLI</li>
-            <li>API Design</li>
-            <li>Web Development</li>
-          </ul>
-        </div>
       </div>
-    </section>
-    <!-- contact form -->
-    <section class="contact-form">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
-      <form action="https://formspree.io/f/xbjezgwz" method="POST">
-        <fieldset>
-          <input
-            type="text"
-            name="user_name"
-            placeholder="Full name"
-            maxlength="30"
-            required
-          />
-          <input type="email" name="user_email" placeholder="Email" required />
-          <textarea
-            name="user_message"
-            maxlength="500"
-            placeholder="Write me something..."
-            required
-          ></textarea>
-          <input
-            class="button"
-            type="submit"
-            name="submit"
-            value="Get in touch"
-          />
-        </fieldset>
-      </form>
-      <img src="Images/Rectangle 42.png" alt="footerdevider" />
-      <ul class="social-media">
-        <li class="social_icon">
-          <a href="https://github.com/rabayarabu"
-            ><img src="Images/Icon -GitHub.png" alt="GitHubIcon"
-          /></a>
-        </li>
-        <li class="social_icon">
-          <a href="https://www.linkedin.com/in/rabaya-rabu-142721169/"
-            ><img src="Images/Icon - Linkedin.png" alt="LinkedinIcon"
-          /></a>
-        </li>
-        <li class="social_icon">
-          <a href=""><img src="Images/Icon - Angelist.png" alt="Angelist" /></a>
-        </li>
-        <li class="social_icon">
-          <a href="https://twitter.com/rabaya_rabu"
-            ><img src="Images/Icon - Twitter.png" alt="TwitterIcon"
-          /></a>
-        </li>
-        <li class="social_icon">
-          <a href=""><img src="Images/Icon - Medium.png" alt="MediumIcon" /></a>
-        </li>
-      </ul>
-      <!-- <img class="footer" src="Images/Light.png" alt="footerBar"> -->
-    </section>
-  </body>
+    </div>
+  </section>
+  <!-- contact form -->
+  <section class="contact-form">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+
+    <form action="https://formspree.io/f/xbjezgwz" method="POST">
+      <fieldset>
+        <input type="text" name="user_name" maxlength="30" placeholder="Full name" required />
+        <input type="email" name="user_email" placeholder="Email" required />
+        <textarea name="user_message" maxlength="500" placeholder="Write me something..."></textarea>
+        <input class="button" type="submit" name="submit" value="Get in touch" />
+      </fieldset>
```

Comment:
- [ ] Dear @rabayarabu, For the submit button we need to follow interactions such as hover, active (when it's pressed), disable, and enable. This is to make sure the user has a pleasant time surfing our page. 

### Find this in the figma template, please.
![image](https://user-images.githubusercontent.com/69077061/230006329-a6e7e2a1-ed29-490b-b4ca-ff351aacbedf.png)